### ISSUE TITLE
vweb include does not output error that the file does not exist

### ISSUE THREAD

**Issue by deadmanwalking74** [vlang/v#9027]
`@include 'nav' // nav.html`

I get this error:

```
$ v -cg run app.v        
V panic: Vweb: Reading file nav failed.
/tmp/v/v2.2117820033329698166.tmp.c:11370: at v_panic: Backtrace
/tmp/v/v2.2117820033329698166.tmp.c:22429: by vweb__tmpl__compile_template
/tmp/v/v2.2117820033329698166.tmp.c:22358: by vweb__tmpl__compile_file
/tmp/v/v2.2117820033329698166.tmp.c:38567: by v__parser__Parser_comp_call
/tmp/v/v2.2117820033329698166.tmp.c:43760: by v__parser__Parser_expr
/tmp/v/v2.2117820033329698166.tmp.c:42046: by v__parser__Parser_expr_list
/tmp/v/v2.2117820033329698166.tmp.c:43237: by v__parser__Parser_return_stmt
/tmp/v/v2.2117820033329698166.tmp.c:41939: by v__parser__Parser_stmt
/tmp/v/v2.2117820033329698166.tmp.c:41535: by v__parser__Parser_parse_block_no_scope
/tmp/v/v2.2117820033329698166.tmp.c:39236: by v__parser__Parser_fn_decl
/tmp/v/v2.2117820033329698166.tmp.c:41617: by v__parser__Parser_top_stmt
/tmp/v/v2.2117820033329698166.tmp.c:41444: by v__parser__Parser_parse
/tmp/v/v2.2117820033329698166.tmp.c:41331: by v__parser__parse_file
/tmp/v/v2.2117820033329698166.tmp.c:41488: by v__parser__parse_files
/tmp/v/v2.2117820033329698166.tmp.c:59457: by v__builder__Builder_gen_c
/tmp/v/v2.2117820033329698166.tmp.c:59486: by v__builder__Builder_build_c
/tmp/v/v2.2117820033329698166.tmp.c:59526: by v__builder__Builder_compile_c
/tmp/v/v2.2117820033329698166.tmp.c:60556: by v__builder__compile
/tmp/v/v2.2117820033329698166.tmp.c:61345: by main__main
/tmp/v/v2.2117820033329698166.tmp.c:61714: by main
```

**Comment by ntrel** [comment]
Could you try to make a minimal example to reproduce this please?

**Comment by fnetX** [comment]
I just ran into the very same issue. Most of the examples also seem to be broken.
This is a minimal setup: https://codeberg.org/eventbike/vlang_plg/src/branch/main/2021-03-25-vweb-include

**Comment by fnetX** [comment]
Oh, sorry for capturing this, I think this thread is about a file not being found? Well, this sounds like a user issue then.

I actually ran into
~~~
V panic: array.get: index out of range (i == 1, a.len == 1)
~~~
with my minimal example.

**Comment by Dialga** [comment]
> Oh, sorry for capturing this, I think this thread is about a file not being found? Well, this sounds like a user issue then.
> 
> I actually ran into
> 
> ```
> V panic: array.get: index out of range (i == 1, a.len == 1)
> ```
> 
> with my minimal example.

That's because you didn't quote your header file
Use `@include 'header'` instead of `@include header`

**Comment by Dialga** [comment]
This issue has been fixed in PR #9052.

`main.v`
```
module main

import vweb

pub struct App { vweb.Context }

fn main() {
	mut app := App{}
	vweb.run<App>(app, 8888)
}

pub fn (mut app App) index() vweb.Result {
	return $vweb.html()
}
```

`index.html`
```
@include 'filenotexist'

<h1>Hello</h1>
```

```
$ v run main.v
index.html:1:1: error: Reading file filenotexist from path: /tmp/index.html failed
    1 | @include 'filenotexist'
      | ~~~~~~~~~~~~~~
    2 |
    3 | <h1>Hello</h1>
```

**Comment by Dialga** [comment]
Closing as fixed.
